CHANGELOG.md

```diff
@@ -84,6 +84,7 @@ Known issues:
 - The Improve This Map tool now uses the same zoom level that is currently being shown in the map view. ([#4068](https://github.com/mapbox/mapbox-gl-native/pull/4068))
 - Fixed a formatting issue in the documentation for `MGLCoordinateBoundsIsEmpty()`. ([#3958](https://github.com/mapbox/mapbox-gl-native/pull/3958))
 - The maximum maximum zoom level is now 21. ([#4417](https://github.com/mapbox/mapbox-gl-native/pull/4417)))
+- Fixed issues with configuration and documentation that caused problems when installing the SDK as a static binary ([#4455](https://github.com/mapbox/mapbox-gl-native/pull/4455))
```

Comment:
Let’s point out that this should also fix App Store submission problems, since the installation itself appears to work without a hitch until you go to release the application.